seedpod_ground_risk/data/external_data_references.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
This can all be moved into `data/__init__.py` to get rid of the `... .external_data_references` package when importing

Reply:
...and update all the existing references

Reply:
By all do you mean moving all of the code in the file or just the importing of os?

Reply:
Moving the entire contents of the file. That way when you try to use these funcs, you will only need to `from seedpod_ground_risk.data import *` instead of `from seedpod_ground_risk.data.external_data_references import *`. The extra module scope is unnecessary